### ISSUE TITLE
A PDF that doesn't read in the right order

### ISSUE THREAD

**Issue by jaybird110127** [trypsynth/paperback#44]
I have a PDF which doesn't read in the right order in Paperback but does according to QRead and the Nao NVDA addon. It's a claim form, and when rendered correctly, items 1 through 23 appear first, but in Paperback items starting with 24 appear, then at the end of the form, the items 1 through 23 appear. Wormhole link: https://wormhole.app/kz4kBb#1dY6wBUdXXz4BpqD9l7SwQ

**Comment by trypsynth** [comment]
Thanks for this, I had seen this myself but hadn't looked into it a ton yet. The file is downloaded and I'll investigate.
